corehq/apps/dashboard/static/dashboard/js/hq_dashboard.ng.js

```diff
@@ -180,6 +180,20 @@
         };
     };
 
+    dashboardDirectives.appsPaginatedTileDirective = function (dashboardConfig) {
```

Comment:
can you think of a way to do this without adding a specific directive for applications? Making a secondary link option generic could potentially be useful for other paginated lists. Allowing the icon class to be specified for the secondary link would help with this.